Makefile

```diff
@@ -19,13 +21,15 @@ TEST_SUITES ?= $(CURDIR)/tests
 # No additional flags by default (used to add --print)
 BAKE_ADDITIONAL_FLAGS ?=
 
+arch := $(shell uname -m)
+
 ##### Macros
 ## Check the presence of a CLI in the current PATH
 check_cli = type "$(1)" >/dev/null 2>&1 || { echo "Error: command '$(1)' required but not found. Exiting." ; exit 1 ; }
 ## Check if a given image exists in the current manifest docker-bake.hcl
 check_image = make --silent list | grep -w '$(1)' >/dev/null 2>&1 || { echo "Error: the image '$(1)' does not exist in manifest. Please check the output of 'make list'. Exiting." ; exit 1 ; }
 ## Generic command to build an image from the manifest
-build_image = docker buildx bake -f docker-bake.hcl --set '*.platform=linux/amd64' --load '$(1)' $(BAKE_ADDITIONAL_FLAGS)
+build_image = set -x; docker buildx bake -f docker-bake.hcl --set '*.platform=linux/$(arch)' --load '$(1)' $(BAKE_ADDITIONAL_FLAGS)
```

Comment:
this probably won't work for building the windows images, but it at least makes it work on other linux platforms

Reply:
yep, as far as I understand the `Jenkinsfile`, the Windows images are build and tested with the Powershell scripts. The Windows agent do not have (yet?) GNU Make neither bash (maybe a cholocatey.git bash or WSL can be a good addon on long term). So good 👍